tcks/microprofile-metrics/rest/pom.xml

```diff
@@ -31,6 +31,9 @@
                     <environmentVariables>
                         <MP_METRICS_TAGS>tier=integration</MP_METRICS_TAGS>
                     </environmentVariables>
+                    <excludes>
+                        <exclude>org.eclipse.microprofile.metrics.test.MpMetricTest</exclude>
```

Comment:
This is almost the whole TCK module. Why?

Reply:
It was only one test failing, due to the Gauge issue, but there's no way to exclude a single test in pom.xml

Reply:
I'm confused how you managed to get that test to fail, I thought this was an issue only for other TCKs that were compiled against Metrics 2.1. Btw I will send a PR to upgrade to Metrics 2.2 as soon as SmallRye Metrics 2.3 gets synced into Central